### PR TITLE
Prod ci pipeline

### PR DESCRIPTION
**Pull request by MarshallAsch**
**Describe the pull request:**

Will automaticly deploy the socis.ca site and a staging site to dev2.socis.ca

---

---

**Checklist:**

- [x] Verify the changes work as expected on at least two browsers
- [x] Verify the changes work as expected on mobile view
- [x] Update documentation / not applicable
- [x] Set yourself as the `Assignee`
- [x] Set `SoorajModi` or `MarshallAsch` as the reviewer

### CODE REVIEW

**Comment by MarshallAsch** [comment]
This did not work, it should have ran a deployment when the review was requested on the PR. 

When do you think the staging  environment should be spun up?

File: .github/workflows/staging.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: Staging
+
+on:
+  pull_request_review:
+    types: [submitted, edited]
```
Comment (SoorajModi) [inline .github/workflows/staging.yml:5]: Try
```
on:
  pull_request:
    branches:
      - main
```

Comment (MarshallAsch) [reply .github/workflows/staging.yml]: updated

**Comment by SoorajModi** [comment]
I think that worked?

**Review by SoorajModi — APPROVED** [review]

**Pull request merged**